source/reference/limitations.txt

```diff
@@ -89,7 +89,7 @@ Sharded Clusters
 - The shard key cannot be :ref:`refined <shard-key-refine>` while
   synchronizing.
 - The shard key cannot be modified using :dbcommand:`reshardCollection`
-  during resharding.
+  during syncing.
```

Comment:
[minor nit]

Consider using "synchronizing" to be more consistent with the copy in the bullet above.

Reply:
Yes, I noticed that too. We use both on this page and across the site. I sent a note to Alan Z asking which we should standardize on. When he responds I'll fix them across the c2c site